packages/core/template/portfolio/contents/skills.md

```diff
@@ -0,0 +1,26 @@
+<frontmatter>
+  title: Skills
+</frontmatter>
+
+<br>
+
+# Skills
+
+This is where you can give a quick overview of the technologies and frameworks you know using our <a href="https://markbind.org/userGuide/components/imagesAndDiagrams.html#thumbnails" target="_blank">thumbnail feature</a> or our <a href="https://markbind.org/userGuide/formattingContents.html#icons" target="_blank">icons</a>. 
+  </p>
+  <panel header="Example using thumbnails" minimized>
+    <box type="info">These thumbnails are available as variables to use anywhere in this page!</box>
+    {{ javascript }}
+    {{ typescript }}
+    {{ vue }}
+  </panel>
+  <br/>
```

Comment:
I think you don't need a linebreak here, it's fine for it to all be on the same line?